src/modules/TodosContainer.js

```diff
@@ -0,0 +1,24 @@
+import Todo from './Todo.js';
+import { getTodos, setTodos } from './store.js';
+import UI from './UI.js';
+
+export default class TodosContainer {
+  todos = getTodos() || [];
+
+  addTodo = (description) => {
+    const id = (Math.floor(Math.random() * 100000)).toString();
+    const index = this.todos.length + 1;
+    const todo = new Todo(id, description, false, index);
+    this.todos.push(todo);
+    setTodos(this.todos);
+    UI.add(id, description);
+  };
```

Comment:
- [ ] I could add an empty string as todo, kindly add a check for that

Reply:
Thanks for the review, I will fix that.